_data/simple-icons.json

```diff
@@ -6438,6 +6438,11 @@
             "hex": "3F5DFF",
             "source": "https://loopback.io/resources"
         },
+        {
+            "title": "Lootcrate",
+            "hex": "111111",
```

Comment:
I cannot find any `#111111` on their website. Could you provide a reference resource?

Reply:
There is no hex source in the website so I used this website https://imagecolorpicker.com/en to get the hex value with the logo. Is this sufficient? If not I could change it to #000000 based on extracting the logo from wikipedia https://en.wikipedia.org/wiki/Loot_Crate